config/src/main/java/net/spals/appbuilder/config/service/ServiceScan.java

```diff
@@ -0,0 +1,78 @@
+package net.spals.appbuilder.config.service;
+
+import com.google.common.base.Predicates;
+import org.inferred.freebuilder.FreeBuilder;
+import org.reflections.Reflections;
+
+import java.util.Arrays;
+import java.util.Set;
+
+/**
+ * Bean for holding configuration for a service scan.
+ *
+ * This is a simple wrapper around a {@link Reflections}
+ * object with syntactic sugar for adding service packages
+ * to the scan.
+ *
+ * @author tkral
+ */
+@FreeBuilder
+public interface ServiceScan {
+
+    Set<String> getServicePackages();
+
+    Reflections getReflections();
```

Comment:
Wrapping `Reflections` in our own `ServiceScan` class. This provides a better development experience when building the underlying `Reflections` scanner.